src/backend/core/roles/roles_manager.py

```diff
@@ -0,0 +1,106 @@
+# Managing roles - groups and permissions
+import json
+from enum import Enum
+from django.contrib.auth.models import Permission, Group
+from .models import ManagerPermission, ManagerGroup
+
+
+class PermissionType(Enum):
+    VIEW = 1
+    ADD = 2
+    CHANGE = 3
+    DELETE = 4
+
+    def __str__(self):
+        return self.name.lower()
+
+    @staticmethod
+    def DjangoPermissionNameToType(name):
```

Comment:
minor comment here: In the rest of the application, we follow [PEP 8](https://peps.python.org/pep-0008/#method-names-and-instance-variables) and use `snake_case` naming for methods. Therefore, I think we should use the same convention for these static methods as well

Reply:
Fair point, I will change it to `snake_case`